raiden/services.py

```diff
@@ -94,6 +105,42 @@ def update_monitoring_service_from_balance_proof(
         )
         return
 
+    # In production there should be no MonitoringRequest if
+    # channel balance is below a certain threshold. This is
+    # a naive approach that needs to be worked on in the future
+    if raiden.config.environment_type == Environment.PRODUCTION:
+        dai_token_network = views.get_token_network_address_by_token_address(
+            chain_state=chain_state,
+            token_network_registry_address=raiden.default_registry.address,
```

Comment:
this is the correct registry_address, right?

Reply:
Yes